cfgov/retirement_api/locale/es/LC_MESSAGES/django.po

```diff
@@ -2,7 +2,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: retirement\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-08-04 13:52-0400\n"
+"POT-Creation-Date: 2023-01-19 11:38-0500\n"
```

Comment:
Hmmm. Let me do some quick digging to see what might be going on here.

So, validate-translations is... imperfect. It simply checks that a newly-generated `.mo` file matches the committed one, to ensure that someone didn't forget to run `compilemessages`. It doesn't compare the generated `.po` files because they (infinite-rolleyes) contain a datetime stamp which makes such a comparison vastly less trivial. So, it's an is-the-commit-consistent-with-what's-generated check.

Because they're already commented out, they must've been commented out and the `.mo` file compiled and committed, so... nothing is changed. Doesn't mean it's correct, but it's consistent.

Reply:
So, this is [because of a known issue with translation-marker-discovery in our Jinja templates with our front-end functional call style](https://cfpb.github.io/consumerfinance.gov/translation/#do-not-include-spaces-between-the-parentheses-and-the-string-in-jinja2-templates). If a `_("foo")` call in a template is instead written as `_( "foo" )` with spaces around the `_()` argument, Django's message scanning with `makemessages` will miss the string. 

It looks like [that each string in that page](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/retirement_api/jinja2/retirement_api/about.html) is like this, but the messages for `retirement_api` weren't regenerated until the change you linked to.

This is an easy fix, but I'll do it in a follow-up PR. I also want to open an issue around how we test for known-bugbears like this.

Reply:
PR here: https://github.com/cfpb/consumerfinance.gov/pull/7466